src/Worlds.js

```diff
@@ -90,3 +137,22 @@ export function isWorldName(str: ?string): boolean {
 export function getWorldProperties(world: WorldName): WorldProperties {
     return worlds[world];
 }
+
+export function getBackgroundInfo(world: WorldName, yPos: number, xPos: number): ?string {
+    const worldBackgroundInfo = worlds[world].backgroundInfo;
+    if (worldBackgroundInfo) {
```

Comment:
I was mentioning making an alternate structure, if you want to be able to omit `backgroundInfo` altogether, then that check would be more like `backgroundInfo && backgroundInfo[yPos] && backgroundInfo[xPos]`.